### ISSUE TITLE
M> Define "Purpose & Principles" for the RChain bounty program (RAM)

### ISSUE THREAD

**Issue by pmoorman** [rchain/bounties#370]
**Outline:**
We notice with new onboarding, that (newly active) members are confused about I) how the RChain coop works, and II) what they can expect, and III) how they can best contribute. 

In issue #253 (and others) the significance of _**cultural education**_ as part of the onboarding came up. Right now this is hard, because no culture code has been defined for RChain.

**Scope:**
To create a standard of "Purpose & Principles", capture it in an easy-to-read document, and make it available for everyone in the coop to use.

**Deliverables**
- [x] Concepting discussions about "Purpose"
- [x] Brainstorm core principles
- [x] Define purpose & principles
- [x] Write the document
- [ ] Community review & improvements
- [ ] Discussion during governance forum (??  I hope)
- [ ] Acceptance of the proposed "Purpose & Principles" document

_NOTE: Execution and integration into the onboarding flow is outside the scope of this issue (at least for now)._ 

**Budget: $5000**

**Comment by pmoorman** [comment]
Hey all, this document has been an internal WIP between me and a few others, but I think it's good to bring it out publicly here.

https://docs.google.com/document/d/1rfpwP3PBVx429ulwxl0ZgalVknV4KIf5sWTdGBXltJQ/edit?usp=sharing

**Summary:**

- We define the purpose of the coop
- We explain — in plain english — the basics of how the coop works
- We outline 5 core values that are essential to anyone getting onboard the coop, or (technically) the "Bounty Program" / RAM

I hope it's just in time for the governance forum, and I'd love to improve upon it based on the discussions that will be held there.

The current document is written by me, and editted / improved primarily by @dckc 

Cheers,

–Pieter

**Comment by pmoorman** [comment]
I should add that this document is now in use as the basis for ["Welcome to the Bounty Program"](https://github.com/rchain/Members/wiki/Welcome-to-the-Bounty-Program) which is a good first step to implementing it!

**Comment by dckc** [comment]
There was a session on purpose at the governance forum today; unfortunately, I neglected to bring this up. Sorry.

Meanwhile, this is one of those tricky issues that goes beyond the scope of the bounty program. We can advise on purpose and principles, but we can't speak for the coop. Then the question is: who are we advising? People are chatting about this in `#exec-committee` so perhaps that's who we should take this proposal to.

Or we can reduce the scope to "purpose and principles of the bounty program."

**Comment by Ojimadu** [comment]
Note aside:
There needs to be a clear picture of the structure of the coop. There seem to be quite some confusion especially when not coming from the board about who speaks for the coop, who approves what, what act can be said to be that of the coop and what act is not. As things stand now, most of the decision making is vested in the board of directors and the president. 
At the 'RAM' meeting of 2018-02-07 @drbloom was of the [thought? opinion? fact?] that 'RAM' was somehow different from the core cooperative. That "RAM" was more of a bounty program put in place by greg and activities here on this repository may/can not be said explicitly to be that of the coop. He likened "RAM" to Reflective ventures and RChain holdings which work closely with the coop but not representative of the coop (That was how I understood the conversation between him and @wordsandstuff (brandon) on the T-shirt for the governance forum). @drbloom is a board member so I don't know if that was just his opinion or a sentiment shared by board, if the latter is true then this needs to be clearly pointed out.
Currently, there is a discussion on the ``` #exec-committee ``` channel on discord on mission statement, vision etc. -reminds me of the recent logo/rebranding talks- that got me wondering peradventure we arrive on a consensus for the mission and vision statements who get to approve them? The board? The committee? This issue #370 also falls into the same question above.

**Comment by dckc** [comment]
>There needs to be a clear picture of the structure of the coop.

I agree; I started an [RChain Organization](https://github.com/rchain/Members/wiki/RChain-Organization) wiki page as a supplement to [Welcome to the Bounty Program](https://github.com/rchain/Members/wiki/Welcome-to-the-Bounty-Program). Is it clear enough? (See also plans to refactor README and CONTRIBUTING in #253.) I'm not sure if the wiki of this repository is the best place for it, but it's convenient for now.

> As things stand now, most of the decision making is vested in the board of directors and the president.

Yes (though there are other officers besides the president.)

> At the 'RAM' meeting of 2018-02-07 @drbloom was of the [thought? opinion? fact?] that 'RAM' was somehow different from the core cooperative.

Is there any reason to doubt that? It's pretty clear from community debriefs in Aug 2017 that the president assigned @lapin7 to run this bounty program, but I haven't seen any action by the board nor officers to equate this forum with the whole coop.

> ... [if] we arrive on a consensus for the mission and vision statements who get to approve them? The board? The committee?

That's the same as my "who are we advising?" question. I'm learning a lot at the governance forum, but I'm still not sure yet. (disclosure / reminder: I'm on the executive committee)

The shortest path is through the officers, but for something as fundamental as this, I expect they'll want to get input from a large portion of the membership through various means.

**Comment by dckc** [comment]
[Aug 30 - RChain Community Debrief 40](https://github.com/rchain/Members/wiki/Weekly-Debrief-Index#-aug-30---rchain-community-debrief-40)
[HJ explains Activist Decentralized Budgeting](https://www.youtube.com/watch?v=7Li4g4qDF6M&t=1486s)

**Comment by pmoorman** [comment]
_@Ojimadu @dckc  I've spun out the point about "clear organisational picture" to it's own issue, because I think it's out of the scope of this issue. **See: #379**_ 

---

@dckc wrote: 

> 
> this is one of those tricky issues that goes beyond the scope of the bounty program. We can advise on purpose and principles, but we can't speak for the coop. Then the question is: who are we advising? People are chatting about this in #exec-committee so perhaps that's who we should take this proposal to.

I didn't want to take this into the #exec-committee, because any discussion there gets flooded with other discussions, and within a week or so effectively all the public memory is erased. So maybe better invite people here to Github (where a clear history management exists)?

> 
> Or we can reduce the scope to "purpose and principles of the bounty program."

In terms of scope: I think the document I've linked to should maybe be scoped down to RAM / bounty instead.  

**But...** before we decide on that, we ought to clarify what exactly even the difference and relationship is between coop/RAM. My suggestion is that maybe we should resolve issue #379 before we resolve this issue here.

**Comment by makys** [comment]
I added two core values to the gdoc.

@pmoorman wrote:

I didn't want to take this into the #exec-committee, because any discussion there gets flooded with other discussions, and within a week or so effectively all the public memory is erased. So maybe better invite people here to Github (where a clear history management exists)?

I would suggest we continue with the momentum and articulate our thoughts on Purpose and Principles of the RAM. However, this may be subject to change as the board and other officers  review this and verifies this is representative of the initial intent.

The way I understand it, the RAM is a subset of the Rchain Coop Members.

**Comment by AyAyRon-P** [comment]
After reviewing the document @pmoorman, I think we could incorporate this entire document into our issue #371. Otherwise we're generating another standalone document when this should be part of the overview/masterbrand platform documentation. 

What're your thoughts on that?

**Comment by dckc** [comment]
Yes, I think merging makes sense, but...

I would be more likely to recognize it by the name "purpose and principles" than "brand platform." Based on discussions at the Governance Forum in Seattle, I think I'm not alone. So I suggest merging #371 into this one.

**Comment by pmoorman** [comment]
@makys I have been incredibly late to respond, but I just wanted to say _"thanks for the feedback / input!"_

Personally I like especially your addition of "raise the flag" which I think points out another principle. Well articulated, too.

@dckc @AyAyRon-P @makys based on all your comments here, **I suggest we change the scope of this issue.** The scope will be towards a focus on RAM/bounty program (which are the same thing). 

That way, this issue will be more leaning towards onboarding, which is what I wrote it for in the first place, anyway. The wider coop is free to decide to jam on this writing, and define purpose & principles along the same line. Maybe I should speak with Greg Meredith or other "founding members" to have a better vantage point towards suggesting purpose & principles for the entire coop.

The "branding platform" serves as a foundation for the branding WG that's currently being formed, and has a more outward-looking / marketing focus.  For that reason, I think things should be separated. 

Can everyone agree with that approach?

I'll change the scope straight away...

**Comment by AyAyRon-P** [comment]
@pmoorman yes I agree that this should refocus on the RAM/Bounty Program and as you mentioned leaning towards the onboarding process. 

Also agree with your comment about the branding platform.

**Comment by dckc** [comment]
We're getting feedback (e.g. PR #419) that [Welcome to the Bounty Program][1] answers this need. The remaining work I see (as I noted elsewhere; IOU link) is to refactor README and CONTRIBUTING to fit with it.

[1]: https://github.com/rchain/bounties/wiki/Welcome-to-the-Bounty-Program

**Comment by pmoorman** [comment]
Yeah, that PR comes basically out of this issue, right? I think right now the "Welcome to the bounty program" page answers the need sufficiently.

Also, I think this document can be taken by the 2 member onboarding WG's (run by @makys and @patrick727 respectively), and be used as an input there.

Regarding the acceptance / governance process:  I think this document has been out there for everyone for quite some time, and also garnered feedback from a quite wide group of people.  maybe it's just "good enough" for now, and can be used in the onboarding efforts going forward.

I suggest -- as a last step -- that I clean up the document a little bit when I have some time, and try integrate all the feedback to the best of my abilities. That way, the resulting document will be easier to use by other people that want to use it later (e.g. the onboarding groups). 

We can leave the issue open for further discussion, because I guess this topic will be reviewed & updated in the future.

**Comment by dckc** [comment]
> The remaining work ... is to refactor README and CONTRIBUTING to fit with it.

done.
